apps/scan/frontend/src/screens/scan_warning_screen.tsx

```diff
@@ -74,20 +72,21 @@ function ConfirmModal({ content, onConfirm, onCancel }: ConfirmModalProps) {
 
 interface MisvoteWarningScreenProps {
   electionDefinition: ElectionDefinition;
+  systemSettings: SystemSettings;
   overvotes: readonly OvervoteAdjudicationReasonInfo[];
   undervotes: readonly UndervoteAdjudicationReasonInfo[];
 }
 
 function MisvoteWarningScreen({
   electionDefinition,
+  systemSettings,
   overvotes,
   undervotes,
 }: MisvoteWarningScreenProps): JSX.Element {
   const returnBallotMutation = returnBallot.useMutation();
   const acceptBallotMutation = acceptBallot.useMutation();
-  const allowCastingOvervotes = !isFeatureFlagEnabled(
-    BooleanEnvironmentVariableName.DISALLOW_CASTING_OVERVOTES
-  );
+  const allowCastingOvervotes =
+    !systemSettings.precinctScanDisallowCastingOvervotes;
```

Comment:
Am I right in thinking that this logic will only come into play if the overvote adjudication reason is turned on? If so, we might want to add some validation to system settings to make sure that you don't turn on `precinctScanDisallowCastingOvervotes` without also having the adjudication reason. Not blocking for this PR though